pacehrh/R/pace_scenarios_manage.R

```diff
@@ -1,41 +1,3 @@
-.scenarioColumnNames <-
-  c(
-    "UniqueID",
-    "WeeksPerYr",
-    "HrsPerWeek",
-    "BaselinePop",
-    "o_PopGrowth",
-    "o_Fertility_decr",
-    "o_MHIVTB_decr",
-    "o_ChildDis_decr",
-    "sheet_TaskValues",
-    "sheet_PopValues",
-    "sheet_SeasonalityCurves",
-    "sheet_Cadre",
-    "DeliveryModel",
-    "Geography_dontedit"
```

Comment:
this column "Geography_dontedit" is still being used in https://github.com/InstituteforDiseaseModeling/PACE-HRH/blob/10ba0ddbf78d4495fd4302c0b9ccfa5d6d4d2b36/pacehrh/R/pace_post_process.R#L209 but this column is removed the demo scenario sheet

Reply:
Also sorted.